apache/trafficcontrol
# TP: Adds topology section to cdn snapshot diff

## What does this PR (Pull Request) do?

Adds a topology section to the cdn snapshot diff view. Also, adds the missing first/inner/last header rewrite columns to all delivery service tables.


## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
- Run the [UI tests](https://github.com/apache/trafficcontrol/tree/master/traffic_portal/test/end_to_end)
- Manually navigate to the cdn snapshot diff view. I.e. https://tp.domain.com/#!/cdns/1/config/changes
- perform a snapshot to clear out the diff.
- add/edit/remove a topology from https://tp.domain.com/#!/topologies
- return to https://tp.domain.com/#!/cdns/1/config/changes and view the diff under the topologies section.
- navigate to any delivery service table and ensure that you can show/hide the first/inner/last header rewrite columns.

## The following criteria are ALL met by this PR
